specification/cognitiveservices/data-plane/ContentModerator/stable/v1.0/ContentModerator.json

```diff
@@ -2512,6 +2512,12 @@
           }
 
         },
+        "SSN": {
```

Comment:
I have no examples, so I can just trust you than the actual JSON received is as described :)

Reply:
I will update the example.

Reply:
I confirm this change is expected.